torch/quantization/observer.py

```diff
@@ -171,40 +197,86 @@ def _calculate_qparams(self, min_val, max_val):
 
 
 class MinMaxObserver(_ObserverBase):
-    r"""Default Observer Module
-    A default implementation of the observer module, only works for
-    `per_tensor_affine` quantization scheme.  The module will record the
-    running average of max and min value of the observed Tensor and
-    calculate_qparams will calculate scale and zero_point
+    r"""Observer module for computing the quantization parameters based on the
+    running min and max values.
+
+    This observer uses the tensor min/max statistics to compute the quantization
+    parameters. The module records the running minimum and maximum of incoming
+    tensors, and uses this statistic to compute the quantization parameters.
+
+    Args:
+        dtype: Quantized data type
+        qscheme: Quantization scheme to be used
+        reduce_range: Reduces the range of the quantized data type by 1 bit
+
+    Given running min/max as :math:`x_\text{min}` and :math:`x_\text{max}`,
+    scale :math:`s` and zero point :math:`z` are computed as:
+
+    The running minimum/maximum :math:`x_\text{min/max}` is computed as:
+
+    .. math::
+
+        x_\text{min/max} = \begin{cases}
+            \min(X) & \text{if~}x_\text{min/max} = \O \\
+            \min(x_\text{min/max}, \min(X)) & \text{otherwise}
+        \end{cases}
+
+    where :math:`X` is the observed tensor.
+
+    The scale :math:`s` and zero point :math:`z` are then computed as:
+
+    .. math::
+
+        \begin{aligned}
+            \text{if Symmetric:}&\\
+            &s = 2 x_\text{max} / \left( Q_\text{max} - Q_\text{min} \right) \\
+            &z = \begin{cases}
+                0 & \text{if dtype is qint8} \\
+                128 & \text{otherwise}
+            \end{cases}\\
+            \text{Otherwise:}&\\
+                &s = \left( x_\text{max} - x_\text{min}  \right ) /
+                    \left( Q_\text{max} - Q_\text{min} \right ) \\
+                &z = Q_\text{min} - \text{round}(x_\text{min} / s)
+        \end{aligned}
```

Comment:
![image](https://user-images.githubusercontent.com/4216323/66856026-2a198280-ef39-11e9-828c-bb273ee0b430.png)

Reply:
For symmetric the equation should be 2.max(x_max, -x_min)/(Q_max -Q_min)

Reply:
use `Symmetric` and `Affine` to align the naming?